be/src/exec/vectorized/schema_scanner.h

```diff
@@ -31,6 +31,7 @@ struct SchemaScannerParam {
     const std::string* ip{nullptr};                   // frontend ip
     int32_t port{0};                                  // frontend thrift port
     int64_t thread_id = 0;
+    int32_t limit{0};
```

Comment:
int32_t is not enough, I think int64_t is OK

Reply:
ok, I will modify it later